### PR TITLE
Feature/python deps

### PR DESCRIPTION
**Pull request by denis-yuen**
**Description**
Remove old unsupported requirements files to remove false positive dependabot hits
Move swagger-ui versioning out to properties file and autogen a requirements file to expose the version to dependabot

**Review Instructions**
Could temporarily update swagger-ui version and build to demonstrate

**Issue**
https://ucsc-cgl.atlassian.net/browse/SEAB-4986
https://ucsc-cgl.atlassian.net/browse/SEAB-4999

Please make sure that you've checked the following before submitting your pull request. Thanks!

- [x] Check that you pass the basic style checks and unit tests by running `mvn clean install`
- [x] Ensure that the PR targets the correct branch. Check the milestone or fix version of the ticket.
- [x] Follow the existing JPA patterns for queries, using named parameters, to avoid SQL injection
- [x] If you are changing dependencies, check the Snyk status check or the dashboard to ensure you are not introducing new high/critical vulnerabilities
- [x] Assume that inputs to the API can be malicious, and sanitize and/or check for Denial of Service type values, e.g., massive sizes
- [x] Do not serve user-uploaded binary images through the Dockstore API
- [x] Ensure that endpoints that only allow privileged access enforce that with the `@RolesAllowed` annotation
- [x] Do not create cookies, although this may change in the future
- [x] If this PR is for a user-facing feature, create and link a documentation ticket for this feature (usually in the same milestone as the linked issue). Style points if you create a documentation PR directly and link that instead.

### CODE REVIEW

File: dockstore-webservice/generated/src/main/resources/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# the original file is dockstore/dockstore-webservice/src/main/resources/requirements/swagger-ui/requirements.template
```
Comment (coverbeck) [inline dockstore-webservice/generated/src/main/resources/requirements.txt:1]: I don't see a requirements.template file.

**Review by coverbeck — COMMENTED** [review]
Disclaimer, I've never looked at how Swagger UI works. I'd assumed there was something going on with Java in the background, but the PR is exposing the Swagger UI PIP package so scanning tools can run on it. 

I'm curious how it works, and why there's a lag the first time you invoke the Swagger UI (what and how is something being built?) -- is Python involved?

It might be easier to do a quick chat than answer my questions in writing :)

File: dockstore-webservice/generated/src/main/resources/requirements.txt (hunk 1 above)
Comment (denis-yuen) [reply dockstore-webservice/generated/src/main/resources/requirements.txt:1]: Ah, forgot to correct comment after I couldn't figure out how to rename the file while filtering

**Comment by denis-yuen** [comment]
> Disclaimer, I've never looked at how Swagger UI works. I'd assumed there was something going on with Java in the background, but the PR is exposing the Swagger UI PIP package so scanning tools can run on it.
> 
> I'm curious how it works, and why there's a lag the first time you invoke the Swagger UI (what and how is something being built?) -- is Python involved?

It's a pip package but the language is mostly Javascript (no idea why)
 https://github.com/swagger-api/swagger-ui 

That said, I think the lag is due to https://dockstore.org/openapi.yaml being slow the first time we hit it

**Review by coverbeck — APPROVED** [review]

**Review by svonworl — APPROVED** [review]

**Review by kathy-t — APPROVED** [review]

**Pull request merged**